ionoscloud/resource_server_test.go

```diff
@@ -68,6 +69,7 @@ func TestAccServerBasic(t *testing.T) {
 				Config: testAccCheckServerNoNicUpdate,
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr(constant.ServerResource+"."+constant.ServerTestResource, "name", constant.ServerTestResource),
+					resource.TestCheckResourceAttr(constant.ServerResource+"."+constant.ServerTestResource, "hostname", constant.ServerTestHostname),
```

Comment:
@cristiGuranIonos since this is the configuration for the update: `testAccCheckServerNoNicUpdate` I suggest using a new `hostname` value for this test, maybe something like `myhostnameupdated`.

Reply:
added, value only "updated"